rmgpy/data/kinetics/family.py

```diff
@@ -2029,7 +2029,7 @@ def getLabeledReactantsAndProducts(self, reactants, products):
         If RMG cannot find appropriate labels, (None, None) will be returned.
         """
         template = self.forwardTemplate
-        reactants0 = [reactant.copy(deep=True) for reactant in reactants]
+        reactants0 = [reactant.molecule[0].copy(deep=True) for reactant in reactants]
         if len(reactants0) == 1:
```

Comment:
Why not catch that before? maybe should add unittests?

Reply:
I think this method is only used in the convert library to training ipynb. With the changes to generateReactionsFromFamilies, this method no longer worked properly. I can add unit tests.